src/v2/Components/ArtworkFilter/ArtworkFilters/FilterExpandable.tsx

```diff
@@ -40,15 +41,22 @@ export const FilterExpandable: React.FC<ExpandableProps> = props => {
         anchorRef.current?.scrollIntoView({ block: "end" })
       }
     }
-  }, [open])
+  }, [open, scrollRef])
 
   const onClick = () => {
-    isScrollIntoViewEnabled && setOpen(open => !open)
+    if (sd.IS_MOBILE) {
```

Comment:
Does this mean that we can only expand filters on mobile web?

Reply:
Not quite like that. We can expand and shrink the filters both on the desktop and on the mobile version. You can find more context about the code in this PR #7707

Reply:
Thanks for sharing the PR - I wasn't able to figure out from the discussion why this check is needed. Could you walk me through this when I sign on tomorrow?

Reply:
This check has been [here](https://github.com/artsy/force/blob/5ff4126690b8a22aa464951604ad9478731aab51/src/v2/Components/ArtworkFilter/ArtworkFilters/FilterExpandable.tsx#L19) before (it was just called differently). It is needed in order to perform scrolling only on mobile devices (watch the attached [gif](https://user-images.githubusercontent.com/44819355/121188349-b21c7080-c871-11eb-8a39-0eae86891853.gif)). 

The reason why I replaced `const isScrollIntoViewEnabled = useMatchMedia (themeProps.mediaQueries.xs)` with `sd. IS_MOBILE` is that `useMatchMedia` works asynchronously => it will not work to make filters hidden by default on mobile devices

Reply:
> it will not work to make filters hidden by default on mobile devices

That makes sense! Thank you :)